src/api/posts.js

```diff
@@ -0,0 +1,16 @@
+const BASE_URL = 'https://mate-api.herokuapp.com';
+
+export function getPostDetails(postID) {
+  return fetch(`${BASE_URL}/posts/${+postID}`)
```

Comment:
```suggestion
  return fetch(`${BASE_URL}/posts/${postID}`)
```
do you need this plus, if result will be converted to the string type?